### PR TITLE
fix multi meta upgrade

### PR DESCRIPTION
**Pull request by critical27**
When multiple meta upgrade:
* leader will upgrade all data from v1 to v2
* follower just wait from leader

https://jira.nebula-graph.io/browse/NG-401

### CODE REVIEW

File: src/meta/MetaVersionMan.h [hunk 1]
```diff
@@ -14,21 +14,28 @@
 namespace nebula {
 namespace meta {
 
+enum class MetaVersion {
+    UNKNOWN = 0,
```
Comment (laura-ding) [inline src/meta/MetaVersionMan.h:18]: why modify to enum, it's hard to expand in the future.

File: src/meta/MetaVersionMan.cpp [hunk 2]
```diff
@@ -330,8 +313,13 @@ Status MetaVersionMan::doUpgrade(kvstore::KVStore* kv) {
             }
         }
     }
-    return Status::OK();
+    if (!setMetaVersionToKV(kv)) {
```
Comment (laura-ding) [inline src/meta/MetaVersionMan.cpp:316]: Not only set the version when upgrade. when the meta has not set the version also need to set the version

**Review by panda-sheep — APPROVED** [review]
Well done！

File: src/meta/MetaVersionMan.cpp (hunk 2 above)
Comment (laura-ding) [reply src/meta/MetaVersionMan.cpp:316]: the 2.0 data couldn't call doUpgrade, the doUpgrade directly to decode by v1

Comment (critical27) [reply src/meta/MetaVersionMan.cpp:316]: OK, I will move it to daemon as before.

File: src/meta/MetaVersionMan.cpp [hunk 3]
```diff
@@ -15,66 +15,49 @@ DEFINE_bool(print_info, false, "enable to print the rewrite data");
...
+        return MetaVersion::UNKNOWN;
     }
     if (iter->valid()) {
         auto v1KeySize = hostPrefix.size() + sizeof(int64_t);
-        if (iter->key().size() != v1KeySize) {
-            return false;
+        if (iter->key().size() == v1KeySize) {
+            return MetaVersion::V1;
+        } else {
+            return MetaVersion::V2;
```
Comment (bright-starry-sky) [inline src/meta/MetaVersionMan.cpp:47]: A little confusion here, is better as below ?
`return (iter->key().size() == v1KeySize) ? MetaVersion::V1 : MetaVersion::UNKNOWN;`

**Review by bright-starry-sky — DISMISSED** [review]
Well done. LGTM.

File: src/meta/MetaVersionMan.cpp (hunk 3 above)
Comment (critical27) [reply src/meta/MetaVersionMan.cpp:47]: Make sense.

**Review by bright-starry-sky — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]
well done！

**Pull request merged**